pkg/libartifact/artifact.go

```diff
@@ -59,6 +59,15 @@ func (a *Artifact) GetDigest() (*digest.Digest, error) {
 	return &artifactDigest, nil
 }
 
+func (a *Artifact) GetManifest() (manifest.OCI1, error) {
+	// TODO Here we have to assume only a single manifest for the artifact; this will
+	// need to evolve
```

Comment:
FYI, talked with @baude about this I will open a PR later to just make it always use one manifest instead of the array as this simplifies a lot of things for all of us I think.
I can do this before your PR here or after, depending on how you prefer it.

Reply:
You can do it before. Please let me know when the PR is ready.

Reply:
Sure I will work on it Monday

Reply:
https://github.com/containers/podman/pull/25200